.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: stable
     hooks:
-    - id: black
-      language_version: python3.6 
+      - id: black
+        language_version: python3.6 
   - repo: https://github.com/pycqa/isort
-    rev: stable
+    rev: 5.10.1
```

Comment:
Minor note: My only concern here is that you're pinning `isort` to `5.10.1` here whereas in the `Pipfile` we take the `*` version from PyPi.

This could mean that once isort 6 is released on PyPi, the two isort verisons (the one used by Pipfile and the one used by pre-commit) will become incompatible and start reporting different suggestions.

It may be worth pinning isort to `~= 5.10` (or `5.*` - same thing) in the Pipfile to make sure the two never get out of sync and start flagging different sort issues